src/k8s/pkg/utils/k8s/status.go

```diff
@@ -2,31 +2,27 @@ package k8s
 
 import (
 	"context"
-	"fmt"
 
 	"github.com/canonical/k8s/pkg/utils/control"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// WaitApiServerReady waits until the kube-apiserver becomes available.
 func (c *Client) WaitApiServerReady(ctx context.Context) error {
```

Comment:
Keep the go doc comment

Reply:
Don't know why I lost all those comments.. sorry 'bout that